packages/components/src/TransactionButton.tsx

```diff
@@ -157,9 +158,8 @@ export class TransactionButtonNoModal extends React.Component<TransactionButtonP
 
   private onClick = async () => {
     const { civil } = this.context;
-
     if (civil && civil.currentProvider) {
-      await civil.currentProviderEnable();
+      // await civil.currentProviderEnable();
```

Comment:
add a comment explaining why this is commented out?

Reply:
this was a mistake - added it as part of testing kirby and missed cleaning it up